src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,156 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> head;
+    private Node<T> tail;
+
     @Override
     public void add(T value) {
+        if (isEmpty()) {
+            createFirstNode(value);
+        } else {
+            Node<T> nextNode = new Node<>(tail, value, null);
+            nextNode.prev.next = nextNode;
```

Comment:
```suggestion
            tail.next = nextNode;
```
we definitely know it's tail